pages/wishlist.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link
+      rel="stylesheet"
+      href="https://dexterous-ui.netlify.app/stylesheets/main.css"
+    />
```

Comment:
You can import this directly in main CSS file.
By this you will not have to link it in every HTML file.